.github/workflows/main.yml

```diff
@@ -68,12 +88,9 @@ jobs:
 
   check:  # This job does nothing and is only used for the branch protection
     if: always()
-
```

Comment:
Please, don't include formatting changes in PRs that aren't dedicated to changing formatting exclusively.